craft_parts/executor/part_handler.py

```diff
@@ -373,6 +373,7 @@ def _run_stage(
             stdout=stdout,
             stderr=stderr,
         )
+        backstage_contents = self._migrate_to_backstage()
```

Comment:
ah, this is interesting. I hadn't thought about what should happen if the user, say, has an `override-stage` without a `craftctl default`. Should the export dir still be migrated?

Reply:
My initial "gut feeling" is that export -> backstage should happen at exactly the same points as install -> stage. So either both happen, or neither happens.

_However_, we apparently always migrate the part's overlay files to stage, so why not do the same with the backstage?

_However however_, the export dir is populated in the same step as the install dir, so conceptually they seem closer and more related (to me).

Reply:
That's a good point. The intention was to make it the same as the migration to `stage` so I'll need to move this.